### DISCUSSION TITLE
Upgrade DB failing in Airflow 2.3.x

### DISCUSSION THREAD

**Discussion by sinjup** [apache/airflow#25974] [Q&A]
Currently I am running Airflow 2.2.5 with python 3.8 on docker. When trying to upgrade to 2.3.x, db upgrade command is not working properly.
**The error I am seeing in the logs - 
airflow-dags-webserver-1   | ERROR: You need to upgrade the database. Please run `airflow db upgrade`. Make sure the command is run using Airflow version 2.3.3.**
Also, my worker exits intermittently, and no other error details regarding that.
How can I fix the issue?

**Comment by potiuk** [comment]
Did you run `airflow db upgrade` as instructed and described in https://airflow.apache.org/docs/apache-airflow/stable/installation/upgrading.html ?

**Reply by github-aman** [reply]
Hi @potiuk , Could you please help me on downgrading Airflow setup running on ECS service as i am getting below error:-

You need to upgrade the database. Please run `airflow db upgrade`. Make sure the command is run using Airflow version 2.6.0

I am at currently Airflow v2.7.0 but need to perform downgrade just to perform rollback testing if something breaks in my app.

I have performed already below in the script which kicks off the Airflow from script.

airflow db upgrade
airflow db downgrade --from-version 2.7.0 --to-version 2.6.0
airflow db migrate

Please suggest on above !!

**Reply by potiuk** [reply]
I think if you have your own issue vaguely related to another discussion/issue, you should open your own discussion. This way maybe *SOMEONE* will help. you if you describe it well enough. People here help when they have free time and feel they can help - not because they are called to help. And by not opening new issue and calling me specifically you limit your chances to get help from others.

So if you seek help - I suggest you to open a new discussion (and hope *someone* will help.

**Reply by github-aman** [reply]
Sure Potiuk, will take care of this & create a new issue.
Apologies for this.

Thanku,
